### DISCUSSION TITLE
Enable Session Recording

### DISCUSSION THREAD

**Discussion by asakth22** [gravitational/teleport#7109] [Q&A]
How do I enable SSH session recording ?
I can see the session recordings under Activity, but I'm not able to play them.
I get a message 'Recording for this session is not available.'

**Comment by webvictim** [comment]
Session recordings should work out of the box. How did you deploy Teleport - did you follow a guide, use Kubernetes/AWS/GCP etc?

**Reply by asakth22** [reply]
I don't get you. 
And I followed this article here https://goteleport.com/docs/getting-started/ to setup the Teleport node.
I've built this node on vCloud and to connect SSH session through teleport proxy which is working really well.
But I don't find a way to record the session.

**Reply by benarent** [reply]
I would double check your roles, try adding the 'auditor' role to view sessions

**Reply by asakth22** [reply]
It looks like we should create a separate user with 'auditor' role to view sessions.
Thanks!!
